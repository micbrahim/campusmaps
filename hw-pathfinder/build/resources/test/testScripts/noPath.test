# Tests Djikstra's algo with no path between nodes

CreateGraph g1

AddNode g1 n1
AddNode g1 n2
AddNode g1 n3

AddEdge g1 n1 n2 6.9

FindPath g1 n1 n3