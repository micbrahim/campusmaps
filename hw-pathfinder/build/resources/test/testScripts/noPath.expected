# Tests Djikstra's algo with no path between nodes

created graph g1

added node n1 to g1
added node n2 to g1
added node n3 to g1

added edge 6.900 from n1 to n2 in g1

path from n1 to n3:
no path found